shared/settings/invites/index.desktop.js

```diff
@@ -201,7 +201,6 @@ function AcceptedInviteItem({
       <Kb.Avatar username={invite.username} size={32} />
       <Kb.Box style={{...Styles.globalStyles.flexBoxColumn, marginLeft: Styles.globalMargins.small}}>
         <Kb.ConnectedUsernames type="BodySemibold" usernames={[invite.username]} />
-        <Kb.Text type="BodySmall">{invite.fullname}</Kb.Text>
```

Comment:
This was a bug masked by a $FlowIssue -- fullname would never be shown and isn't in the RPC payload.  I thought about starting to fix it and then realized that we don't even use account invites anymore, so I think it can stay as-is.